components/mysql/actions/find-row/find-row.mjs

```diff
@@ -0,0 +1,59 @@
+import commonTable from "../common/table.mjs";
+
+const { mysql } = commonTable.props;
+
+export default {
+  ...commonTable,
+  key: "mysql-find-row",
+  name: "Find Row",
+  description: "Finds a row in a table via a lookup column. [See the docs here](https://dev.mysql.com/doc/refman/8.0/en/select.html)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    ...commonTable.props,
```

Comment:
Since you are importing the prop for `app`. I suggest you change the property description here. The actual description is **The database table to watch for changes**. I guess something more specific would be great. Something like: **Table to find row** or something like this